src/raylib.h

```diff
@@ -1226,6 +1226,8 @@ RLAPI Rectangle GetCollisionRec(Rectangle rec1, Rectangle rec2);
 // NOTE: This functions do not require GPU access
 RLAPI Image LoadImage(const char *fileName);                                                             // Load image from file into CPU memory (RAM)
 RLAPI Image LoadImageRaw(const char *fileName, int width, int height, int format, int headerSize);       // Load image from RAW file data
+RLAPI Image LoadImageSvg(const char* fileName);                                                          // Load image from SVG file data with default size from SVG
+RLAPI Image LoadImageSvgWithSize(const char* fileName, int width, int height);                                   // Load image from SVG file data with specified size
```

Comment:
This function can be just renamed to `LoadImageSvg()` for now but not sure what I'll do with it because disabling SVG support would break it, raylib does not expose any file-format-specific loading function, tries to keep that data internal always... but I can review it later.

Reply:
I've renamed LoadImageSvgWithSize() to LoadImageSvg() and added LoadImageSvgFromString as a helper function to work from both LoadImageSvg() and LoadImage(). This has the added side bonus of allowing one to construct their own svg strings and have these loaded as images.

Reply:
I think this function is overengineering, probably not many users (if any) will need this function. I prefer to avoid it.